dali/pipeline/pipeline.cc

```diff
@@ -554,32 +568,52 @@ void Pipeline::SetCompletionCallback(ExecutorBase::ExecutorCallback cb) {
   executor_->SetCompletionCallback(std::move(cb));
 }
 
+bool Pipeline::ValidateOutputs(const DeviceWorkspace &ws) const {
+  DALI_ENFORCE(
+      ws.NumOutput() == static_cast<int>(output_descs_.size()),
+      make_string("Inconsistent output description. Number of outputs does not match. Expected: ",
```

Comment:
Is this _Inconsistent output description_ though? I support that term in the error saying you specified different number of types than there are outputs, but here it was confusing to me. I'd drop the first sentence from the three error messages here.

Reply:
Done